balance: Баланс лута в косморуинах

vovakons opened the pull request:
## Что этот ПР делает

1. Удалено выпадение синди снайпы и пилы.
2. Вместо c20r теперь выпадает ослабленная версия: 50% урона.
3. Вместо m90 выпадает ослабленная версия: 50% урона и разряженный подствольник.
4. Вместо арбалета выпадает ослабленная версия: увеличенный разброс

## Почему это хорошо для игры

Нерф дизбаланс лута с косморуин.

## Тестирование

Локалка.

vovakons committed:
rusted variant of c-20r and m90 guns for syndiedepot loot

Добавлены ослабленные версии для црки и р-90 для лута в синдидепоте.
Урон порезан на 50% и у р-90 стартово разряжен подствольный граник.
Удалена снайпа в луте.

vovakons committed:
Merge branch 'master220' into ruins-loot-balance

dageavtobusnick commented:
Чет какой-то не интересный нерф. Ты же хотел механ сделать.

vovakons commented:
> Чет какой-то не интересный нерф. Ты же хотел механ сделать.

да, хочу сделать отдельным пром выделение rusted пушек через компонент, плюс сделаю компонент для осечек. А это чисто балансный пр

dageavtobusnick commented:
> > Чет какой-то не интересный нерф. Ты же хотел механ сделать.
> 
> да, хочу сделать отдельным пром выделение rusted пушек через компонент, плюс сделаю компонент для осечек. А это чисто балансный пр

Ну значит подождем компонента, а уже потом будем балансить.

vovakons commented:
> Ну значит подождем компонента, а уже потом будем балансить.

😭

vovakons commented:
ожидает мержа пра https://github.com/ss220-space/Paradise/pull/8049

vovakons committed:
Merge branch 'master220' into ruins-loot-balance

vovakons committed:
add rusted and misfire elements

Добавление элементов осечек и ржавости для ганов с руин: c-20r/rusted и
m90/rusted

vovakons commented:
готов к мержу 😎

dageavtobusnick commented:
С учетом растед и осечек урон мод уж сишком маленький. Минимум 0.85. А в остальном норм

vovakons committed:
rusted guns damage mod fix

vovakons commented:
поменял на 0.85
